src/seedu/addressbook/data/person/Name.java

```diff
@@ -40,7 +40,7 @@ public static boolean isValidName(String test) {
      * Retrieves a listing of every word in the name, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        return Arrays.asList(fullName.toLowerCase().split("\\s+"));
```

Comment:
Instead of converting everything to lower case in this method itself, can you find another place that's better to do the conversion?(what if you need to get the words in its original cases later on?)